Userland/Libraries/LibGfx/BitmapFont.cpp

```diff
@@ -348,6 +349,20 @@ String BitmapFont::qualified_name() const
     return String::formatted("{} {} {}", family(), presentation_size(), weight());
 }
 
+String BitmapFont::variant() const
+{
+    StringBuilder variant;
+    variant.append(weight_to_name(weight()));
+    if (slope()) {
+        if (variant.to_string() == "Regular")
```

Comment:
No need to create a new string, just
```suggestion
        if (variant.string_view() == "Regular"sv)
```